javarosa/src/main/java/org/javarosa/core/util/SizeBoundVector.java

```diff
@@ -34,26 +30,14 @@ public int getAdditional() {
     }
 
     public void addBadImageReference() {
-        badImageReferenceCount++;
+        // TODO: remove this
```

Comment:
Ok, I think it probably makes more sense to revert the changes to this file for now than leaving it in a half-but-not-fully-deleted state